backend/src/db/migrations/20230927100749_update_user_achievements_description.ts

```diff
@@ -0,0 +1,220 @@
+      inProgress:
+        'You are actively engaging with discussions! Your participation is valuable, and you are well on your way to achieving this milestone.',
+      done: 'Congratulations on leaving your fifth comment! 🎉 You are becoming a respected voice in the community. Keep those insights flowing!',
+    },
+  },
+  {
+    key: 'write_10_comments',
+    description: {
+      notStarted: NOT_STARTED_COMMENT_ACHIEVEMENT_DESCRIPTION,
+      earlyStage:
+        'You are making great strides in the world of commenting! Keep engaging with the community!',
+      inProgress:
+        'You are actively engaging with discussions and making an impact! Keep it up, and you will achieve this milestone in no time.',
+      done: 'Congratulations on leaving your tenth comment! 🎉 Your contributions are valued, and you are a key part of the community.',
+    },
+  },
+  {
+    key: 'write_15_comments',
+    description: {
+      notStarted: NOT_STARTED_COMMENT_ACHIEVEMENT_DESCRIPTION,
+      earlyStage:
+        'You are on your way to writing fifteen comments! Keep engaging with the community!',
+      inProgress:
+        'You are actively participating in discussions and making your voice heard!',
+      done: 'Congratulations on leaving your fifteenth comment! 🎉 Your contributions are invaluable, and you are a respected member of the community.',
+    },
+  },
+  {
+    key: 'write_25_comments',
+    description: {
+      notStarted: NOT_STARTED_COMMENT_ACHIEVEMENT_DESCRIPTION,
+      earlyStage:
+        'You are actively engaging with discussions and contributing to the community! Your dedication is taking you closer to this achievement.',
+      inProgress: 'You are well on your way to leaving twenty-five comments!',
+      done: 'Congratulations on leaving your twenty-fifth comment! 🎉 You are a valuable member of the community, and your insights are making a difference.',
+    },
+  },
+  {
+    key: 'write_50_comments',
+    description: {
+      notStarted: NOT_STARTED_COMMENT_ACHIEVEMENT_DESCRIPTION,
+      earlyStage:
+        'You are making substantial contributions to discussions! Keep sharing your insights.',
+      inProgress:
+        'You are actively participating in discussions and leaving your mark! Your dedication to commenting is paying off as you work toward this achievement.',
+      done: 'Congratulations on leaving your fiftieth comment! 🎉 You are a prolific commenter and a cornerstone of the community. Keep the conversations alive!',
+    },
+  },
+];
+
+const TABLE_NAME = 'achievements';
+
+const ColumnName = {
+  ID: 'id',
+  KEY: 'key',
+  NAME: 'name',
+  DESCRIPTION: 'description',
+  CREATED_AT: 'created_at',
+  UPDATED_AT: 'updated_at',
+} as const;
+
+async function up(knex: Knex): Promise<void> {
+  for (const achievement of UPDATED_ACHIEVEMENTS) {
+    await knex(TABLE_NAME)
+      .where(ColumnName.KEY, achievement.key)
+      .update({ description: JSON.stringify(achievement.description) });
```

Comment:
Let's not pass stringified JSON to string column - we can make description as jsonb column and store json directly